### PR TITLE
Merge swerve code & dev auto balance

### PR DESCRIPTION
**Pull request by SeanErn**

### CODE REVIEW

**Review by SeanErn — COMMENTED** [review]
Looks good, but don't approve  it until I can test it on BOTH robots

File: src/main/java/frc/robot/ChargedUp/ChargeStation/Cmd_AutoBalance.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    /**
+     * This code will check the angle that the robot is at and drive in the direction that is needed to balance the robot
+     */
+
+    // Get the pitch and roll from the Pigeon IMU.
+    double pitch = m_PigeonGyro.getRawGyroPitch();
+    double roll = m_PigeonGyro.getRawGyroRoll();
+    
+    // If pitch is greater than 8 or less than -8...
+    if (pitch > 8 || pitch < -8) {
```
Comment (Rkortlandt) [inline src/main/java/frc/robot/ChargedUp/ChargeStation/Cmd_AutoBalance.java:42]: Remove useless comment explaining the if statement replace 8 & -8 with constants like maxPitch and minPitch

File: src/main/java/frc/robot/ChargedUp/ChargeStation/Cmd_AutoBalance.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        // Drive using the pitch as the speed. (gyro angle * 0.01 if the gyro is 8 degrees off, the robot will drive at 0.08 speed)
+        m_DriveTrain.Drive(-pitch * 0.01, 0, 0, false, false, false);
+    } else if (roll > 8 || roll < -8) { // If roll is greater than 8 or less than -8...
```
Comment (Rkortlandt) [inline src/main/java/frc/robot/ChargedUp/ChargeStation/Cmd_AutoBalance.java:45]: Remove useless comment. Replace 8 & -8 with constants.

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -53,32 +59,58 @@ public class RobotContainer {
...
+    /*
     MecanumDriveSubSys.setDefaultCommand(
       new Cmd_MecanumDriveDefault(
         MecanumDriveSubSys, 
       () -> m_driverController.getLeftX(), 
       () -> m_driverController.getLeftY(),
       () -> m_driverController.getRightX())
       ); 
+
+      */
```
Comment (Rkortlandt) [inline src/main/java/frc/robot/RobotContainer.java:101]: Move to bottom or somewhere out of the way

File: src/main/java/frc/robot/RobotContainer.java [hunk 4]
```diff
@@ -53,32 +59,58 @@ public class RobotContainer {
...
         driveSubSys,
         () -> driverStation.DriveFwdAxis(),
         () -> driverStation.DriveStrAxis(),
         () -> driverStation.DriveRotAxis(),
-        true,
+        false,
         () -> driverStation.RotateLeftPt(),
         () -> driverStation.RotateRightPt()));
-    */
+*/
```
Comment (Rkortlandt) [inline src/main/java/frc/robot/RobotContainer.java:114]: This too

**Review by Rkortlandt — CHANGES_REQUESTED** [review]
Good over all just some formatting things in the new command. It would be helpful if we moved the old drive code to the bottom.

File: src/main/java/frc/robot/RobotContainer.java (hunk 3 above)
Comment (SeanErn) [reply src/main/java/frc/robot/RobotContainer.java:101]: I don't know if it works yet so I just left it there.

File: src/main/java/frc/robot/ChargedUp/ChargeStation/Cmd_AutoBalance.java (hunk 1 above)
Comment (SeanErn) [reply src/main/java/frc/robot/ChargedUp/ChargeStation/Cmd_AutoBalance.java]: Fixed with new commit

File: src/main/java/frc/robot/RobotContainer.java (hunk 4 above)
Comment (SeanErn) [reply src/main/java/frc/robot/RobotContainer.java:114]: I don't know if it works yet

File: src/main/java/frc/robot/RobotContainer.java (hunk 3 above)
Comment (Rkortlandt) [reply src/main/java/frc/robot/RobotContainer.java:101]: ok

**Review by Rkortlandt — APPROVED** [review]

File: src/main/java/frc/robot/ChargedUp/ChargeStation/Cmd_AutoBalance.java [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    /**
+     * This code will check the angle that the robot is at and drive in the direction that is needed to balance the robot
+     */
+
+    // Get the pitch and roll from the Pigeon IMU.
+    double pitch = m_PigeonGyro.getRawGyroPitch();
+    double roll = m_PigeonGyro.getRawGyroRoll();
```
Comment (Rkortlandt) [inline src/main/java/frc/robot/ChargedUp/ChargeStation/Cmd_AutoBalance.java:40]: Is this comment really necessary?

Comment (SeanErn) [reply src/main/java/frc/robot/ChargedUp/ChargeStation/Cmd_AutoBalance.java:40]: fixed

**Comment by SeanErn** [comment]
Waiting for path planner code

**Pull request closed without merging**